src/ui/components/layouts/base_layout.rs

```diff
@@ -19,30 +19,31 @@ pub fn BaseLayout() -> impl IntoView {
 
   view! {
     <div class="drawer h-full">
-      <input id="mobile-drawer" type="checkbox" class="drawer-toggle"/>
+      <input id="mobile-drawer" type="checkbox" class="drawer-toggle" />
       <div class="drawer-content h-full min-h-screen md:min-h-0 md:max-h-screen flex flex-col">
         <Transition>
           <Unpack item=theme let:theme>
-            <Html attr:data-theme=theme class="h-full max-h-screen overflow-y-hidden"/>
-            <TopNav/>
+            <Html attr:data-theme=theme class="h-full max-h-screen overflow-y-hidden" />
+            <TopNav />
             <div class="flex gap-x-4 h-full overflow-y-auto md:overflow-y-hidden grow">
-              <aside class="w-fit px-3.5 whitespace-nowrap pb-5 pt-3 border-e border-neutral hidden md:flex flex-col bg-base-300 overflow-y-auto h-full">
-
-                <SideNav/>
-              </aside>
+              <nav
```

Comment:
`aside` tags are meant to enhance another attribute while being related to it, but not neccessary to understand it. Since this is a site-wide nav, the `nav` tag is more appropriate.